components/dashboard/sections/collectives/common.tsx

```diff
@@ -35,9 +36,24 @@ export const cols: Record<string, ColumnDef<any, any>> = {
       };
       const collective = row.original;
       const children = mapValues(groupBy(collective.childrenAccounts?.nodes, 'type'), 'length');
-      const secondLine = toPairs(children)
-        .map(([type, count]) => count && `${count} ${formatCollectiveType(intl, type, count)}`)
-        .join(', ');
+      const isChild = collective.parent;
+      const secondLine = isChild ? (
```

Comment:
If we need to be able to sort by parent name, we might as well make `parent` a column, wdyt?

Reply:
Perhaps, a parent column would potentially complicate the table though, as in having to consider whether to dynamically add that column when including projects and events to prevent showing two columns with the same info in the default case.

Since the results are sorted from the API, we could still add a sorting option that is parent collective name, without a column.